plugins/modules/aws_acm.py

```diff
@@ -157,6 +162,26 @@
     choices: [present, absent]
     default: present
     type: str
+
+  tags:
+    description:
+      - Tags to apply to certificates imported in ACM.
+      - >
+        If both I(name_tag) and the 'Name' tag in I(tags) are set,
+        the values must be the same.
+      - >
+        If the 'Name' tag in I(tags) is not set and I(name_tag) is set,
+        the I(name_tag) value is copied to I(tags).
+    type: dict
+    version_added: 3.1.0
+
+  purge_tags:
+    description:
+      - whether to remove tags not present in the C(tags) parameter.
+    default: false
+    type: bool
+    version_added: 3.1.0
```

Comment:
```suggestion
    version_added: 3.2.0
```

...if we get another review